docker-compose.yml

```diff
@@ -29,4 +29,15 @@ services:
       - $HOME/$TRAVIS_BUILD_ID/build:/opt/module/build
     depends_on:
       - standalone-chrome
-    command: 'npm run wdio-default'
\ No newline at end of file
+    command: 'npm run wdio-default'
+
+  test-ci-lowlight:
```

Comment:
We don't need to add and create additional theme services in the docker compose files. The fusion theme didn't need to override the default command for the test-ci service, however, if we need to for this theme, we can. I image they should align, but you could override like:

```bash
 script: travis_wait docker-compose run -e FORM_FACTOR=enormous test-ci -sh 'npm run wdio wdio-lowlight'
```

Reply:
Addressed it here : https://github.com/cerner/terra-core/pull/2758/commits/fd620ecc89460da9ff35c88045eda70a913c3d82